src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_wrapper_windows_inner.cc

```diff
@@ -19,13 +19,13 @@
 #include <grpc/support/port_platform.h>
 
 #include "src/core/lib/iomgr/port.h"
-#if GRPC_ARES == 1 && (defined(GRPC_UV) || defined(GPR_WINDOWS))
+#if GRPC_ARES == 1 && defined(GRPC_ARES_RESOLVE_LOCALHOST_MANUALLY)
```

Comment:
nit: rename this file from `grpc_ares_wrapper_windows_inner.cc` to `grpc_ares_wrapper_localhost.cc` ?